### PR TITLE
GPU: Print each GPU on a different line

### PR DESCRIPTION
**Pull request by dylanaraps**
## Description

This PR makes `get_gpu()` display each GPU on a separate line on Linux systems. 

## Features

- Each GPU is printed on a separate line.

## Issues

- None yet.

## TODO

- [x] Testing
- [x] Add option to print `all`, `dedicated` or `integrated`.

CC @konimex,, can you test this?

### CODE REVIEW

**Comment by konimex** [comment]
It works!

2016-12-22-085505_804x536_scrot

For some reason, battery subtitle seems to be broke, though.

**Comment by dylanaraps** [comment]
Is the battery subtitle broken in master?

**Comment by konimex** [comment]
Yep, seems to be broken in master too.

**Comment by dylanaraps** [comment]
Fixed.

File: neofetch [hunk 1]
```diff
@@ -965,35 +965,51 @@ get_cpu_usage() {
...
+                    ;;
+                esac
+
+                if [[ "$gpu_brand" == "off" ]]; then
+                    gpu="${gpu/AMD }"
+                    gpu="${gpu/NVIDIA }"
+                    gpu="${gpu/Intel }"
+                fi
+
+                prin "GPU${gpu_num}" "$gpu"
```
Comment (konimex) [inline neofetch]: You may want to change the `GPU` one into `${subtitle}`, like the battery function.

Comment (dylanaraps) [inline neofetch]: Yup, just did. :)

Comment (konimex) [inline neofetch]: That was fast....

**Comment by konimex** [comment]
Also, it seems we're still printing the results starting from `GPU0`, should we add the value by 1 so we can start it from `GPU1`?

**Comment by dylanaraps** [comment]
Sure, easy fix. :)

**Comment by dylanaraps** [comment]
I've added `--gpu_type` which lets you pick `dedicated`, `integrated` or `all`. (`all` by default). Please test it out. :)

**Comment by konimex** [comment]
LGTM :+1:

**Pull request merged**